bcda/cclf/cclf_test.go

```diff
@@ -197,7 +197,7 @@ func (s *CCLFTestSuite) TestImportCCLF8() {
 
 	beneficiaries := []models.CCLFBeneficiary{}
 	db.Find(&beneficiaries, "file_id = ?", file.ID)
-	assert.Equal(6, len(beneficiaries))
+	assert.Equal(5, len(beneficiaries))
```

Comment:
How come the number of beneficiaries decreased? Instead of removing `203031406M`, can add a duplicate?